test/sequential/test-module-loading.js

```diff
@@ -127,10 +127,6 @@ assert.equal(require('../fixtures/registerExt2').custom, 'passed');
 assert.equal(require('../fixtures/foo').foo, 'ok',
              'require module with no extension');
 
-assert.throws(function() {
```

Comment:
@Trott Do we really need that? There will be no reference to that in code, but there will be a test which will check if `paths` is `undefined`.